src/pages/Auth/Login.js

```diff
@@ -28,12 +79,22 @@ export const Login = () => {
             <Link to="/">Forgot password?</Link>
           </div>
 
-          <button className="btn-default  btn-primary login-signup-btn">
+          <button

```

Comment:
If you are using form then need not put onClick for a button to submit. Instead make the button's type=submit and in the form tag put onSubmit={submitFormHandler}

In the submit form , put e.preventDefault() to prevent the page reload after form submit

Reply:
Done [f7266a7](https://github.com/Manpreet-hub/Tunes/pull/3/commits/f7266a706e8d7abc8221261fa933c96298fb7da8)